pyrasterframes/python/setup.py

```diff
@@ -118,14 +118,17 @@ def run(self):
     license='Apache 2',
     setup_requires=['pytest-runner', pyspark_ver, 'pathlib'],
     install_requires=[
+        'pytz', 'shapely',
+        # 'geomesa-pyspark'
```

Comment:
Didn't work?

Should we go ahead and add numpy, or should we wait until we add support in `TileUDT`?

Reply:
I would wait to add numpy until we actually depend on it.

Reply:
And correct because not on pypi this does not work to install geomesa-pyspark